cpp/src/bindings/bindings_helper.cpp

```diff
@@ -30,6 +30,32 @@ extern "C"
 		    fit, alignment, *availableBounds, *artboardBounds, *requiredBounds);
 	}
 
+	JNIEXPORT void JNICALL
+	Java_app_rive_runtime_kotlin_core_Helpers_cppConvertToArtboardSpace(
+	    JNIEnv* env,
+	    jobject thisObj,
+	    jlong touchSpaceBoundsRef,
+	    jlong touchSpaceLocationRef,
+	    jobject jfit,
+	    jobject jalignment,
+	    jlong artboardSpaceBoundsRef,
+	    jlong artboardSpaceLocationRef)
+	{
+		auto fit = ::getFit(env, jfit);
+		auto alignment = ::getAlignment(env, jalignment);
+		auto artboardSpaceBounds = (rive::AABB*)artboardSpaceBoundsRef;
+		auto touchSpaceBounds = (rive::AABB*)touchSpaceBoundsRef;
+		auto touchSpaceLocation = (rive::Vec2D*)touchSpaceLocationRef;
+		auto artboardSpaceLocation = (rive::Vec2D*)artboardSpaceLocationRef;
+
+		rive::Mat2D forward = rive::computeAlignment(
+		    fit, alignment, *touchSpaceBounds, *artboardSpaceBounds);
+		rive::Mat2D inverse = forward.invertOrIdentity();
+
+		rive::Vec2D convertedLocation = inverse * *touchSpaceLocation;
+		*artboardSpaceLocation += convertedLocation;
```

Comment:
curious: why are we adding instead of just assigning?

Reply:
Yeah, this function could return a `Float[2]`, not sure if JNI makes it a nightmare though.
Do we care about any other value other than `artboardSpaaceLocation`?